src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+import model.Ball;
+
+public class Lottery {

```

Comment:
The class 'Lottery' should have a private constructor to prevent instantiation if all methods inside are static. However, if the getRandomBall method is not meant to be static, then it's fine to have a public constructor.